### PR TITLE
Create CI

### PR DESCRIPTION
**Pull request by raphaklaus**
Issue: #3

Use GitHub actions to build, fetch and test the library.

### CODE REVIEW

File: .github/workflows/elixir.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Elixir CI
```
Comment (cabol) [inline .github/workflows/elixir.yml]: Maybe I'd do some small changes:

```
name: CI

on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]

jobs:
  mix_test:
    name: mix test (Elixir ${{ matrix.elixir }} OTP ${{ matrix.otp }})
    strategy:
      matrix:
        elixir: ['1.9.4', '1.10.2']
        include:
          - elixir: '1.9.4'
            otp: '22.x'
          - elixir: '1.10.2'
            otp: '22.x'
    runs-on: ubuntu-16.04
    steps:
      - uses: actions/checkout@v1
      - uses: actions/setup-elixir@v1
        with:
          otp-version: ${{ matrix.otp }}
          elixir-version: ${{ matrix.elixir }}
      - name: Install Dependencies
        run: mix deps.get
      - name: Run Tests
        run: mix test
```

Comment (raphaklaus) [reply .github/workflows/elixir.yml]: It looks good. But why Ubuntu 16.04?

Comment (raphaklaus) [reply .github/workflows/elixir.yml]: Also, I think we can trigger the CI on any branch as it will enable to run the tests for feature/fix branches.

Comment (cabol) [reply .github/workflows/elixir.yml]: Sure, make sense, so let's keep the `ubuntu-latest` you had, and also enable for all branches `on: [pull_request, push]`

Comment (raphaklaus) [reply .github/workflows/elixir.yml]: I've put just `pull_request` event as `pull_request, push` was triggering double CIs.

**Pull request merged**